powershell-adapter/Tests/TestClassResource/0.0.1/TestClassResource.psm1

```diff
@@ -74,6 +74,49 @@ class TestClassResource : BaseTestClass
     }
 }
 
+[DscResource()]
+class NoExport: BaseTestClass
+{
+    [DscProperty(Key)]
+    [string] $Name
+
+    [DscProperty()]
+    [string] $Prop1
+
+    [DscProperty()]
+    [string] $EnumProp
+
+    [void] Set()
+    {
+    }
+
+    [bool] Test()
```

Comment:
for this `NoExport` class I would get rid of all not-used business-logic (`if` conditions) in `Test` and `Get` methods.

Reply:
will update